apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -37,10 +38,17 @@ export const mapVehicle = (type: ?string, uniqueNo: ?string) => ({
   uniqueNo: uniqueNo ?? null,
 });
 
-export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => ({
-  name: null, // @TODO - value is currently missing from API endpoint response
-  code: routeItem?.airline ?? null,
-});
+export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => {
+  // @TODO - value is currently missing from API endpoint response
+  // JSON file is used from https://github.com/BesrourMS/Airlines/blob/master/airlines.json
+  const airlineCode = routeItem?.airline;
+  const airline = airlines.find(airline => airline.iata === airlineCode);
+  const name = airline?.name;
```

Comment:
The tests are failing because when there is no `name` the expression returns undefined (and the test expects null). You can either change the test or `airline?.name ?? null`

Reply:
Sorry :(
I will update test as airline name also got updated

Reply:
Or would be better if I just update airline name part?

Reply:
You don't need to do `airline?.name ?? null` since optional chaining already returns null or undefined. `airline?.name` should be enough

Reply:
@tbergq In that case the test should be updated because if there is an airline with an undefined name it returns undefined and the test was failing because it expected null

Reply:
then we should update the test of course

Reply:
And test updated :)